jina/clients/base/stream_rpc.py

```diff
@@ -0,0 +1,68 @@
+from jina.clients.helper import callback_exec
+from jina.proto import jina_pb2_grpc
+
+
+class StreamRpc:
+    """Class that encapsulated the methods required to run a stream rpc call from the client. Instantiate a single class
+    for each client request.
+    """
+
+    def __init__(
+        self,
+        channel,
+        continue_on_error,
+        metadata,
+        on_always,
+        on_done,
+        on_error,
+        p_bar,
+        req_iter,
+        max_attempts,
+        backoff_multiplier,
+        initial_backoff,
+        max_backoff,
+        logger,
+        show_progress,
+        compression,
+        **kwargs
+    ):
+        self.compression = compression
+        self.show_progress = show_progress
+        self.logger = logger
+        self.max_backoff = max_backoff
+        self.initial_backoff = initial_backoff
+        self.backoff_multiplier = backoff_multiplier
+        self.max_attempts = max_attempts
```

Comment:
The `max_attempts` would not take effect, right? I'm not so sure that is it reasonable to remind the user `max_attemps` is not considered in stream rpc.